plugins/server-features/node-modules.ts

```
@@ -0,0 +1,90 @@
+        case "csv":
+        case "html":
+        case "xml":
+        case "json":
+            return MakeTextRes(manager, nodeModuleFile);
+
+        case "ts":
+        case "tsx":
+        case "jsx":
+        case "js":
+        case "css":
+            const formatedExt = ext == "css" ? "css" : "js";
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)